index.html

```diff
@@ -4,26 +4,70 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Logoipsum</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="script.js" defer></script>
 </head>
 <body>
-    <header>
-        <img src="" alt="">
-        <nav>
-            <ul>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
-                <li><a href=""></a></li>
-            </ul>   
-        </nav>
-        <button type="button">Get started</button>
-    </header>
-    <main>
-        <h1>Слепенков Игорь</h1>
-    </main>
-    <footer>
+    <div class="container">
+        <header>
+            <div class="header__logo__wrap">
+                <img class="header__logo" src="logo.png" alt="Logo">
+            </div>
 
-    </footer>
+            <nav>
+                <ul>
+                    <li><a href="">Home</a></li>
+                    <li><a href="">Features</a></li>
+                    <li><a href="">Pricing</a></li>
+                    <li><a href="">Blog</a></li>
+                </ul> 
+            </nav>
+
+            <div class="header__button__wrap">
+                <button type="button">Get started</button>

```

Comment:
you should write naming class block__element-modifyer

Reply:
Fixed